connect/runtime/src/test/java/org/apache/kafka/connect/runtime/errors/RetryWithToleranceOperatorTest.java

```diff
@@ -222,25 +224,31 @@ public void testExecAndHandleRetriableErrorWithMaxRetriesExceeded() throws Excep
 
     public void execAndHandleRetriableError(long errorRetryTimeout, int numRetriableExceptionsThrown, List<Long> expectedWaits, Exception e, boolean successExpected) throws Exception {
         MockTime time = new MockTime(0, 0, 0);
-        CountDownLatch exitLatch = PowerMock.createStrictMock(CountDownLatch.class);
+        CountDownLatch exitLatch = mock(CountDownLatch.class);
         RetryWithToleranceOperator retryWithToleranceOperator = new RetryWithToleranceOperator(errorRetryTimeout, ERRORS_RETRY_MAX_DELAY_DEFAULT, ALL, time, new ProcessingContext(), exitLatch);
         retryWithToleranceOperator.metrics(errorHandlingMetrics);
 
-        EasyMock.expect(mockOperation.call()).andThrow(e).times(numRetriableExceptionsThrown);
-
-        if (successExpected) {
-            EasyMock.expect(mockOperation.call()).andReturn("Success");
-        }
+        when(mockOperation.call()).thenAnswer(new Answer<String>() {
+            private int count = 0;
+
+            @Override
+            public String answer(InvocationOnMock invocation) throws Throwable {
+                if (count++ != numRetriableExceptionsThrown) {
+                    throw e;
+                } else if (successExpected) {
+                    return "Success";
+                }
+                return null;
+            }
+        });
```

Comment:
We can simplify this a bit:
```suggestion
        OngoingStubbing<String> mockOperationCall = when(mockOperation.call());
        for (int i = 0; i < numRetriableExceptionsThrown; i++) {
            mockOperationCall = mockOperationCall.thenThrow(e);
        }
        if (successExpected) {
            mockOperationCall.thenReturn("Success");
        }
```

Reply:
Ah neat, I hadn't considered doing it this way. It does look cleaner 👍